.github/workflows/book.yml

```diff
@@ -13,9 +13,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install mdbook
-        uses: peaceiris/actions-mdbook@v1
-        with:
-          mdbook-version: "0.3.7"
```

Comment:
alternatively, we could do:

```
   uses: peaceiris/actions-mdbook@v1
         with:
           mdbook-version: "latest"
```

But, I'm not sure what we're getting from using this action vs. using cargo, it just seems like complexity overkill.

Reply:
Ah, nice! Let's try this out, yeah. I used the other action previously to speed up deployment -- no compiling mdbook. But I guess that other action isn't keeping up to date